### PR TITLE
More Pixel/AOSP stuff

### PR DESCRIPTION
**Pull request by JoshAtticus**
Title ^

### CODE REVIEW

**Comment by Blitz517** [comment]
"com.google.android.apps.diagnosticstool" may be recommended?

**Comment by JoshAtticus** [comment]
> "com.google.android.apps.diagnosticstool" may be recommended?

Removing it breaks repair mode as well so probably not

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -37236,5 +37236,59 @@
     "dependencies": [],
     "neededBy": [],
     "labels": []
+  },
+  {
+    "id": "com.android.DeviceAsWebcam",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:37241]: Are you sure it's with capital letters?

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -37236,5 +37236,59 @@
...
+    "list": "Oem",
+    "description": "Allows using certain Pixel phones as a USB webcam.\n Will break USB function picker.",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:37243]:
```suggestion
    "description": "Allows using certain Pixel phones as a USB webcam.\nWill break USB function picker.",
```
Remove redundant space.

File: resources/assets/uad_lists.json [hunk 3]
```diff
@@ -37236,5 +37236,59 @@
...
+    "description": "Required for most AI features on Google Pixel devices such as magic eraser, best take, now playing etc.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.google.android.apps.diagnosticstool",
+    "list": "Oem",
+    "description": "Google Pixel diagnostics tool.\n Exclusive to Pixel Watch 1 & 2 along with Pixel 6 series and later running Wear OS 4/Android 14.",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:37279]:
```suggestion
    "description": "Google Pixel diagnostics tool.\nExclusive to Pixel Watch 1 & 2 along with Pixel 6 series and later running Wear OS 4/Android 14.",
```
Remove redundant space.

File: resources/assets/uad_lists.json [hunk 4]
```diff
@@ -37236,5 +37236,59 @@
...
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.google.android.apps.dreamliner",
+    "list": "Oem",
+    "description": "Google Pixel AI wallpaper.\n Exclusive to Pixel 6 series and later running Android 14.",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:37288]:
```suggestion
    "description": "Google Pixel AI wallpaper.\n Exclusive to Pixel 6 series and later running Android 14.",
```
Remove redundant space.

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Comment by AnonymousWP** [comment]
@JoshAtticus Please follow-up on the above.

File: resources/assets/uad_lists.json (hunk 1 above)
Comment (JoshAtticus) [reply resources/assets/uad_lists.json:37241]: Yes it is

**Comment by AnonymousWP** [comment]
> > @JoshAtticus Please follow-up on the above.
> 
> It won't let me commit any of the changes because the diff has changed or whatever

In that case, just make the changes I suggested and push to your fork/branch so it's included in this PR.

**Comment by AnonymousWP** [comment]
Why did you close this PR??

**Comment by JoshAtticus** [comment]
> Why did you close this PR??

I don't know how to do that stuff

**Comment by AnonymousWP** [comment]
@Blitz517 Could you please add some Pixel/OASP packages from this current PR but via a new PR?

**Comment by JoshAtticus** [comment]
> @AnonymousWP these apps looks like a from android 14, I will check if its possible

Yes, they are from Android 14

**Pull request closed without merging**